Components/FacetsSection/FacetsSection.js

```diff
@@ -17,60 +20,84 @@ const ContentSection = styled(Section)`
   padding: 0 136px;
   display: flex;
   flex-direction: column;
+  position: relative;
 `;
 
-const FacetSection = styled.button`
-  background-color: transparent;
+const FacetSection = styled.div`
+  background-color: ${getColorValue('grey')};
   margin-bottom: 32px;
-  border-color: ${getColorValue('grey')};
-  display: flex;
-  align-items: center;
-  justify-content: space-between;
-`;
-
-const FacetImage = styled.img`
-  height: 150px;
-  width: 40%;
-  border: 2px solid ${getColorValue('grey')};
-  margin-top: 54px;
-  margin-bottom: 54px;
-  margin-left: 32px;
+  clip-path: ${props => props.clipPath};
 `;
 
 const FacetSectionTitle = styled(H3)`
   padding: 54px 32px;
-  width: 60%;
-  display: flex;
-  flex-wrap: wrap;
+`;
+
+const Link = styled(BaseLink)`
   justify-content: center;
 `;
 
 const Title = styled(H1)`
   text-align: center;
-  
-`
+`;
 
 export const FacetsSection = () => {
+  const [isVisible, makeVisible] = useState(null);
+  const [top, setAlertTopValue] = useState(null);
+
+  useEffect(() => {
+    const timer = setTimeout(() => {
+      makeVisible(false);
+    }, 3000);
+    return () => clearTimeout(timer);
+  }, [isVisible]);
+
+  const showAlert = (e) => {
+    e.preventDefault();
+    makeVisible(true);
+    setAlertTopValue('5%');
+  }
+
+  const showAlert2 = (e) => {
+    e.preventDefault();
+    makeVisible(true);
+    setAlertTopValue('35%');
+  }
+
+  const showAlert3 = (e) => {
+    e.preventDefault();
+    makeVisible(true);
+    setAlertTopValue('65%');
+  }
+  
   return (
-    <ParentSection>
+    <ParentSection id="facets">
       <ContentSection>
-        <FacetSection>
-          <FacetImage src={'https://via.placeholder.com/150'} />
-          
-          <FacetSectionTitle>{'Portfolio, Experiences, and Skills'}</FacetSectionTitle>
-        </FacetSection>
-
-        <FacetSection>
-          <FacetImage src={'https://via.placeholder.com/150'} />
+        <Wrapper>
+          <FacetSection clipPath={'polygon(20% 0%, 80% 0%, 100% 20%, 100% 80%, 80% 100%, 20% 100%, 0% 80%, 0% 20%)'}>
```

Comment:
since these are all the same shape, this doesn't need to be a property anymore... but if i like the new designs that are in my head, i'd merge as is and change the design instead of updating this now...